scripts/create-component/plop-templates/src/components/{{componentName}}/{{componentName}}.types.ts.hbs

```diff
@@ -1,7 +1,8 @@
-import type { ComponentProps, ComponentState, IntrinsicShorthandProps } from '@fluentui/react-utilities';
+import type { ComponentProps, ComponentState, IntrinsicSlotProps } from '@fluentui/react-utilities';
 
 export type {{componentName}}Slots = {
-  root: IntrinsicShorthandProps<'div'>;
+  // TODO Add slots here
```

Comment:
why the TODO ?

Reply:
Hotell Martin Hochel FTE 9 hours ago Member

why the TODO ?

 
 khmakoto Makoto Morimoto Burgos (he/him) FTE Author Member Pending

Oops, I think this was a bad merge, thanks for noticing!